### PR TITLE
Add all known Kourend & Kebos diary requirements

### PR DESCRIPTION
**Pull request by Monsterxsync**
Adds all the currently known diary requirements will update this when the levels i'm missing are known to us, was also thinking about adding the house favour requirements to the plugin as there are many that require them.

### CODE REVIEW

**Comment by Nightfirecat** [comment]
Unless we know the exact text ahead of time (I don't think we will?) I think this should probably wait to be released the week after, or sometime shortly after the game update as a point release perhaps.

**Comment by Abextm** [comment]
They showed the complete list in game on stream, so I doubt the wording will change too much

**Comment by Monsterxsync** [comment]
diary reqs image Heres an image with the requirements i doubt they would change the wording around. You never know though

File: runelite-client/src/main/java/net/runelite/client/plugins/achievementdiary/diaries/KourendDiaryRequirement.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+		add("Create a Strength potion in the Lovakengi Pub.",
+			new SkillRequirement(Skill.HERBLORE, 12));
+		add("Fish a Trout from the River Molch.",
+			new SkillRequirement(Skill.FISHING, 20));
+
+		//MEDIUM
+		add("Travel to the Fairy Ring south of Mount Karuulm.",
+			new QuestRequirement(Quest.FAIRYTALE_II__CURE_A_QUEEN, true));
+		add("Use Kharedst's memoirs to teleport to all five cities in Great Kourend.",
+			new QuestRequirement(Quest.THE_DEPTHS_OF_DESPAIR, false),
```
Comment (Spedwards) [inline runelite-client/src/main/java/net/runelite/client/plugins/achievementdiary/diaries/KourendDiaryRequirement.java]: Adding `false` to the quest requirements is not required and is redundant as it already defaults that way.

**Comment by Abextm** [comment]
Added in e0534ebb22f3bf9120c89ba4d2fe86916b68a62a

**Pull request closed without merging**